doc/topics/installation/windows.rst

```diff
@@ -37,9 +38,21 @@ The 64bit installer has been tested on Windows 7 64bit and Windows Server
 Please file a bug report on our GitHub repo if issues for other platforms are
 found.
 
-The installer asks for 2 bits of information; the master hostname and the
-minion name. The installer will update the minion config with these options and
-then start the minion.
+The installer will detect previous installations of Salt and ask if you would
+like to remove them. Clicking OK will remove the Salt binaries but should leave
```

Comment:
`should leave`? :]

I'd want to know for certain if my configs and keys were going to be deleted. ;]